### ISSUE TITLE
Example of selecting option value using select step?

### ISSUE THREAD

**Issue by kensoh** [kelaberetiv/TagUI#455]
from user query in https://github.com/kelaberetiv/TagUI/issues/454#issuecomment-503546722

---

One more issue which I was facing was in the UI, I had a dropdown and based on the documentation I tried a few things but then the option value is not getting selected.

```
select //*[@id="abc1"]/div/div/div/div[1]/div/div/div/div[3]/div[1]/div/span[1]/div/select/ as Travelling 
And even tried specifying the option number instead.
select //*[@id="abc1"]/div/div/div/div[1]/div/div/div/div[3]/div[1]/div/span[1]/div/select/ as 2 or select //*[@id="abc1"]/div/div/div/div[1]/div/div/div/div[3]/div[1]/div/span[1]/div/select/ as option[2]
```

The documentation didn't have many examples so was a little hard to follow. Can you help me out here?

Thanks.

**Comment by kensoh** [comment]
@abhishek-27, first double check that the XPath element identifier is correct. You can do that by searching in Chrome web inspector, or trying to do a click action on the element with TagUI. Second is inspect the dropdown to see the value attribute of the option. TagUI uses the HTML value attribute of the option item as it is usually the unique identifier for the option. After that use that value as the second parameter.

Alternatively, you can consider writing custom JavaScript code and use dom step to send the JS code to browser to do the selection.

Or consider using visual automation to visually click on the dropdown menu and visually selecting the option you want. This way, don't have to use XPath or web selectors.

**Comment by abhishek-27** [comment]
Yeah i got it working with dom step and query selector like this: 

`dom document.querySelector("#abc1 > div > div > div > div.insidePage_wrapper > div > div > div > div:nth-child(3) > div.apply_grid_container > div > span.apply_grid_col1 > div > select > option:nth-child(2)").setAttribute('selected', true)`

**Comment by kensoh** [comment]
Cool! wow this looks like a tough to locate web element, nested deep within the anchor tag 👍
